data-manipulation/encoding/base64/decode-data-using-base64.yml

```
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: decode data using Base64
```

```suggestion
    name: decode data using Base64 via dword translation table
```

updating name to differentiate against the other base64 rule we have (https://github.com/fireeye/capa-rules/blob/master/data-manipulation/encoding/base64/encode-data-using-base64.yml)

---

will need to change filename to reflect this new name, too